### PR TITLE
Round 2 of basic cleaning for JTA code

### PR DESCRIPTION
**Pull request by arjantijms**
Signed-off-by: arjantijms <arjan.tijms@gmail.com>

### CODE REVIEW

File: appserver/transaction/jts/src/main/java/com/sun/enterprise/transaction/jts/iiop/TransactionServerInterceptor.java [hunk 1]
```diff
@@ -16,18 +17,20 @@
...
+import org.omg.PortableInterceptor.ServerRequestInfo;
+import org.omg.PortableInterceptor.ServerRequestInterceptor;
 
 import com.sun.enterprise.transaction.api.JavaEETransactionManager;
 
-public class TransactionServerInterceptor extends LocalObject
-        implements ServerRequestInterceptor, Comparable {
+public class TransactionServerInterceptor extends LocalObject implements ServerRequestInterceptor, Comparable {
 
+    /**
```
Comment (pzygielo) [inline appserver/transaction/jts/src/main/java/com/sun/enterprise/transaction/jts/iiop/TransactionServerInterceptor.java:30]: Please, remove this unnecessary comment.

File: appserver/transaction/jts/src/main/java/com/sun/enterprise/transaction/jts/recovery/RecoveryLockFile.java [hunk 2]
```diff
@@ -24,36 +25,36 @@
...
  *
- * @see
- * Records in the recovery lock file have the following format:
- * PREFIX INSTANCE_NAME TIMESTAMP
- * Where PREFIX can be one of:
- * - "O" means OWNED by this instance, i.e. non-delegated recovery
- * - "B" means recovered BY the specified instance
- * - "F" means recovered FOR the specified instance
- * TIMESTAMP is the time of the recovery operation
+ * @see Records in the recovery lock file have the following format: PREFIX INSTANCE_NAME TIMESTAMP Where PREFIX can be
```
Comment (pzygielo) [inline appserver/transaction/jts/src/main/java/com/sun/enterprise/transaction/jts/recovery/RecoveryLockFile.java:53]: Broken list formatting.

File: appserver/transaction/jts/src/main/java/com/sun/enterprise/transaction/jts/iiop/TransactionServerInterceptor.java (hunk 1 above)
Comment (arjantijms) [reply appserver/transaction/jts/src/main/java/com/sun/enterprise/transaction/jts/iiop/TransactionServerInterceptor.java]: thx!

File: appserver/transaction/jts/src/main/java/com/sun/enterprise/transaction/jts/recovery/RecoveryLockFile.java (hunk 2 above)
Comment (arjantijms) [reply appserver/transaction/jts/src/main/java/com/sun/enterprise/transaction/jts/recovery/RecoveryLockFile.java]: Fixed

File: appserver/transaction/jts/src/main/java/com/sun/enterprise/transaction/jts/iiop/TxIORInterceptor.java [hunk 3]
```diff
@@ -16,33 +17,36 @@
...
 import org.omg.PortableInterceptor.IORInfo;
 import org.omg.PortableInterceptor.IORInterceptor;
 
 import com.sun.logging.LogDomains;
-import org.glassfish.enterprise.iiop.api.GlassFishORBHelper;
-import org.glassfish.hk2.api.ServiceLocator;
 
 public class TxIORInterceptor extends LocalObject implements IORInterceptor {
 
+    /**
```
Comment (pzygielo) [inline appserver/transaction/jts/src/main/java/com/sun/enterprise/transaction/jts/iiop/TxIORInterceptor.java:44]: Please, remove this unnecessary comment.

File: appserver/transaction/jts/src/main/java/com/sun/enterprise/transaction/jts/iiop/TransactionClientInterceptor.java [hunk 4]
```diff
@@ -20,70 +21,85 @@
...
 import org.omg.PortableInterceptor.ClientRequestInterceptor;
-import com.sun.corba.ee.spi.presentation.rmi.StubAdapter;
 
+import com.sun.corba.ee.spi.presentation.rmi.StubAdapter;
 import com.sun.enterprise.transaction.api.JavaEETransactionManager;
 
 public class TransactionClientInterceptor extends LocalObject
         implements ClientRequestInterceptor, Comparable<TransactionClientInterceptor> {
 
+    /**
```
Comment (pzygielo) [inline appserver/transaction/jts/src/main/java/com/sun/enterprise/transaction/jts/iiop/TransactionClientInterceptor.java:31]: Please, remove this unnecessary comment.

**Pull request merged**